### PR TITLE
Web/referral markup

### PR DESCRIPTION
**Pull request by NikolaiYurchenko**
What: 
 Referral markup 
Why:
to implement referral markup and logic

Please describe the tests:
 - Test 1:
 - Test 2:
 
Please describe the performance impact:

## Code Review Checklist (to be filled out by reviewer)
 - [ ] Does the PR describe what changes are being made?
 - [ ] Does the PR describe why the changes are being made?
 - [ ] Does the code follow [our style guide](https://github.com/storj/docs/blob/master/code/Style.md)?
 - [ ] Does the code follow [our testing guide](https://github.com/storj/docs/blob/master/code/Testing.md)?
 - [ ] Is the PR appropriately sized? (If it could be broken into smaller PRs it should be)
 - [ ] Does the new code have enough tests? (*every* PR should have tests or justification otherwise. Bug-fix PRs especially)
 - [ ] Does the new code have enough documentation that answers "how do I use it?" and "what does it do?"? (both source documentation and [higher level](https://github.com/storj/docs), diagrams?)
 - [ ] Does any documentation need updating?
 - [ ] Do the database access patterns make sense?

### CODE REVIEW

File: web/satellite/src/components/account/referral/ReferralArea.vue [hunk 1]
```diff
@@ -0,0 +1,397 @@
...
+                </div>
+                <div class="icon-container">
+                    <img src="../../../../static/images/referral/money.svg">
+				    <p class="referral-container__icons__description">You get XGb in free credits</p>
+                </div>
+			</div>
+			<div class="referral-container__copy-and-share-container">
+				<p class="referral-container__copy-and-share-container__title">Copy And Share Your Invitation Link</p>
+				<div class="referral-container__copy-and-share-container__link-holder">
+					<p class="referral-container__copy-and-share-container__link-holder__link">test://test/test/test/test</p>
```
Comment (VinozzZ) [inline web/satellite/src/components/account/referral/ReferralArea.vue]: FYI, this is the logic for referral link: https://github.com/storj/storj/blob/bdcb40fbc8b64eb315980cfe17ace5af804df6b4/satellite/rewards/partners.go#L78.

File: web/satellite/src/api/referral.ts [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+export async function fetchReferralInfo(): Promise<RequestResponse<ReferralInfo>> {
+    let result: RequestResponse<ReferralInfo> = new RequestResponse<ReferralInfo>();
+
+    let response: any = await apollo.query(
+        {
+            query: gql(`
+                query {
+                    activeReward (
+                        type: 1,
```
Comment (VinozzZ) [inline web/satellite/src/api/referral.ts]: Can we make this type info to be dynamic and then create a constant value to hold the type info? That way, we can use this query to query activeReward for other types too

File: web/satellite/src/components/account/referral/ReferralArea.vue [hunk 3]
```diff
@@ -0,0 +1,412 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+<template>
+	<div class="referral-container">
+		<NoProjectState v-if="isNoProjects"/>
+		<div class="referral-container__wrapper" v-if="!isNoProjects">
+			<p class="referral-container__title">Refer A Friend And You'll Both Get Free Credits</p>
+			<p class="referral-container__subtitle">You Get X GB, They Get X GB</p>
```
Comment (VinozzZ) [inline web/satellite/src/components/account/referral/ReferralArea.vue]: I think the `X` here should be dynamic and display the value from the active reward

File: web/satellite/src/components/account/referral/ReferralArea.vue [hunk 4]
```diff
@@ -0,0 +1,412 @@
...
+			<div class="referral-container__icons">
+                <div class="icon-container">
+				    <img src="../../../../static/images/referral/send.svg">
+                    <p class="referral-container__icons__description">Copy and share your invite link</p>
+                </div>
+                <div class="icon-container">
+                    <img src="../../../../static/images/referral/coins.svg">
+				    <p class="referral-container__icons__description">They get XGB in free credits</p>
```
Comment (VinozzZ) [inline web/satellite/src/components/account/referral/ReferralArea.vue]: Same as above

File: web/satellite/src/components/account/referral/ReferralArea.vue [hunk 5]
```diff
@@ -0,0 +1,412 @@
...
+                </div>
+                <div class="icon-container">
+                    <img src="../../../../static/images/referral/calendar.svg">
+				    <p class="referral-container__icons__description">They use our network for 60 days</p>  
+                </div>
+                <div class="icon-container">
+                    <img src="../../../../static/images/referral/money.svg">
+				    <p class="referral-container__icons__description">You get XGb in free credits</p>
```
Comment (VinozzZ) [inline web/satellite/src/components/account/referral/ReferralArea.vue]: same here

File: web/satellite/src/components/account/referral/ReferralArea.vue (hunk 3 above)
Comment (NikolaiYurchenko) [reply web/satellite/src/components/account/referral/ReferralArea.vue]: Thought its a static value based on organization politics

File: web/satellite/src/utils/apolloManager.ts [hunk 6]
```diff
@@ -9,7 +9,8 @@ import { AuthToken } from '@/utils/authToken';
 
 // Satellite url
 const satelliteUrl = new HttpLink({
-    uri: '/api/graphql/v0',
+    // TODO: delete before push
```
Comment (VinozzZ) [inline web/satellite/src/utils/apolloManager.ts]: Left TODO here

File: web/satellite/src/types/referral.d.ts [hunk 7]
```diff
@@ -0,0 +1,11 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+declare type ReferralInfo = {
```
Comment (VinozzZ) [inline web/satellite/src/types/referral.d.ts]: I think we also want `InviteeCredit` and `InviteeCreditDurationDays` too

File: web/satellite/src/components/account/referral/ReferralArea.vue (hunk 3 above)
Comment (VinozzZ) [reply web/satellite/src/components/account/referral/ReferralArea.vue]: @stoweandi is checking on this one

File: web/satellite/src/types/referral.d.ts (hunk 7 above)
Comment (VinozzZ) [reply web/satellite/src/types/referral.d.ts]: It should be `InviteeCreditInCents`

**Pull request closed without merging**